src/Controller/Api/CategoryController.php

```diff
@@ -31,9 +31,13 @@
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 /**
+ * NEXT_MAJOR: Remove this file.
+ *
  * @final since sonata-project/classification-bundle 3.14
  *
  * @author Vincent Composieux <vincent.composieux@gmail.com>
+ *
+ * @deprecated since sonata-project/classification-bundle 3.x, to be removed in 4.0.
```

Comment:
Can you please remove the `@final` annotation

Reply:
Done